release-team/role-handbooks/docs/Release-Timeline.md

```diff
@@ -444,6 +444,10 @@ If you need help reviewing PRs, ask SIG Docs (#sig-docs channel on Slack) for he
 
 If no suitable candidates are available, you may choose to lead again or nominate someone outside of the current release team (e.g: a recent Docs Lead).
 
+## Late-Middle Steps (Weeks 7-8) - Release Blog Progress and SIG Docs (~4 weeks before the release)
+
+Reach out to the Communications and Enhancements Teams about the progress on the Release Blog.
```

Comment:
wondering if this needs to be more specific? bc i can't really tell what this is^^ are you expecting feedback from those teams? or just telling them that you started? what is the purporse is my question i guess.

Reply:
@kikisdeliveryservice I put the PR on hold for now. This will be flushed out as we define how Docs should work with Comms, Enhancements, and Release Notes e.g. any of the teams can relay to Release Notes about any enhancements or deprecations that will be good for the Major Themes section of the release notes